src/style.css

```diff
@@ -0,0 +1,156 @@
+:root {
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --background-button-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
```

Comment:
The card width should be 200px, including border. Change width to 200px and adjust padding accordingly.

Reply:
just add `box-sizing: border-box;`